imports/plugins/core/ui/client/components/link/link.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+/**
+ * @file
+ * Link is a drop-in replacement for the HTML <a> tag. It uses Reaction's router to navigate to the specified href,
+ * preventing the entire client app from reloading. It also scrolls to the top of the next page.
+ *
+ * @module Link
+ * @extends Component
+ */
+
+export default class Link extends Component {
+  static propTypes = {
+    href: PropTypes.string.isRequired
+  };
+
+  handleClick = (event) => {
+    event.preventDefault();
+    this.props.onClick(event);
+    ReactionRouter.go(this.props.href); // eslint-disable-line no-undef
+    window.scrollTo(0, 0);
```

Comment:
In order to get smooth scrolling in newer browsers and guard against errors in tests or older browsers, maybe update this to:

```js
if (typeof window !== "undefined" && typeof window.scrollTo === "function") {
  window.scrollTo({ left: 0, top: 0, behavior: "smooth" });
}
```